apps/api/cmd/flow/main_test.go

```
@@ -0,0 +1,74 @@
+	defer server.Close()
+
+	binary := filepath.Join(t.TempDir(), "flow")
+	build := exec.CommandContext(t.Context(), "go", "build", "-o", binary, ".")
+	if output, err := build.CombinedOutput(); err != nil {
+		t.Fatalf("build flow: %v: %s", err, output)
+	}
+	input := filepath.Join(t.TempDir(), "create.json")
+	if err := os.WriteFile(input, []byte(`{"type":"x","working_title":"Signal recovery","status":"draft","content":{"body":"body"}}`), 0o600); err != nil {
+		t.Fatal(err)
+	}
+	command := exec.Command(binary, "content", "create", "--file", input, "--json")
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)